src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2737,15 +2737,16 @@ void CalcPurchAirLoads(EnergyPlusData &state,
 
         } // Cooling or heating required
 
-        // EMS override point  Purch air supply temp and humidty ratio ..... but only if unit is on, SupplyMassFlowRate>0.0
-        if (PurchAir(PurchAirNum).EMSOverrideSupplyTempOn) {
-            PurchAir(PurchAirNum).SupplyTemp = PurchAir(PurchAirNum).EMSValueSupplyTemp;
-        }
-        if (PurchAir(PurchAirNum).EMSOverrideSupplyHumRatOn) {
-            PurchAir(PurchAirNum).SupplyHumRat = PurchAir(PurchAirNum).EMSValueSupplyHumRat;
-        }
-
         if (SupplyMassFlowRate > 0.0) {
+            // EMS override point  Purch air supply temp and humidty ratio ..... but only if unit is on, SupplyMassFlowRate>0.0
+            if (PurchAir(PurchAirNum).EMSOverrideSupplyTempOn) {
+                PurchAir(PurchAirNum).SupplyTemp = PurchAir(PurchAirNum).EMSValueSupplyTemp;
+            }
+            if (PurchAir(PurchAirNum).EMSOverrideSupplyHumRatOn) {
+                PurchAir(PurchAirNum).SupplyHumRat = PurchAir(PurchAirNum).EMSValueSupplyHumRat;
+            }
+            SupplyEnthalpy = PsyHFnTdbW(PurchAir(PurchAirNum).SupplyTemp, PurchAir(PurchAirNum).SupplyHumRat);
+
```

Comment:
I don't see how setting the EMS conditions all the time to only when mass flow > 0 changes the answer. But then there are only 2 changes here so these changes certainly did something. Maybe it was the 2nd change that fixed this and this change just updates based on EMS only when there is flow. I guess that's OK (see other comment).